K2Field.K2NE.ServiceBroker/ServiceObjects/Client API/OutOfOfficeClientSO.cs

```diff
@@ -46,10 +46,20 @@ public override List<ServiceObject> DescribeServiceObjects()
             addOutOfOffice.ReturnProperties.Add(Constants.SOProperties.OutOfOffice.DestinationUser);
             so.Methods.Add(addOutOfOffice);
 
+            Method removeOutOfOffice = Helper.CreateMethod(Constants.Methods.OutOfOfficeClient.RemoveOutOfOffice, "Remove user share (destination user) from OOF Shares for a current user.", MethodType.Read);
+            removeOutOfOffice.InputProperties.Add(Constants.SOProperties.OutOfOffice.DestinationUser);
+            removeOutOfOffice.Validation.RequiredProperties.Add(Constants.SOProperties.OutOfOffice.DestinationUser);
+            removeOutOfOffice.ReturnProperties.Add(Constants.SOProperties.OutOfOffice.DestinationUser);
+            so.Methods.Add(removeOutOfOffice);
+
             Method listUserShares = Helper.CreateMethod(Constants.Methods.OutOfOfficeClient.ListUserShares, "Get all the destination users for OOF shares of current user", MethodType.List);
             listUserShares.ReturnProperties.Add(Constants.SOProperties.OutOfOffice.DestinationUser);
             so.Methods.Add(listUserShares);
 
+            Method removeAllShares = Helper.CreateMethod(Constants.Methods.OutOfOfficeClient.RemoveAllShares, "Remove All Shares", MethodType.Execute);
+            removeAllShares.ReturnProperties.Add(Constants.SOProperties.OutOfOffice.CallSuccess);
```

Comment:
I don't see this property being filled anywhere else in the change request.

I would suggest NOT to provide a 'success or not' output. If it doesn't work, just throw an exception.